jib-gradle-plugin/src/integration-test/resources/projects/war_jetty_servlet25/build.gradle

```diff
@@ -12,18 +12,18 @@ repositories {
 }
 
 configurations {
-  moreLibs
+    moreLibs
```

Comment:
nit: I think we should actually change all these test projects' build file indentation to 2 spaces for consistency with our other build files that use 2 spaces for indentation.

Reply:
Took this opportunity to fix other files too.